bin/katello-certs-check

```diff
@@ -165,7 +171,7 @@ check-ca-bundle
 check-cert-san
 check-cert-usage-key-encipherment
 
-if [[ $EXIT_CODE == "0" ]] && [[ $CERT_HOSTNAME == $HOSTNAME ]]; then
+if [ $EXIT_CODE == "0" -a "$TARGET" == "foreman" ]; then
```

Comment:
I'm more fan of the bash conditions since they also handle unset parameters properly so they're a bit safer.
```suggestion
if [[ $EXIT_CODE == "0" ]] && [[ $TARGET == "foreman" ]]; then
```